packages/wallet/test/jest.config.js

```diff
@@ -2,6 +2,7 @@ const { pathsToModuleNameMapper } = require('ts-jest/utils')
 const { compilerOptions } = require('./tsconfig')
 
```

Comment:
We could have a shared config files directory in monorepo root and either use those files directly (if no changes needed and it doesn't need to be under package root), or extending those files. Similarly to how tsconfig works right now.

Reply:
Will also defer this to another PR since it touches the `cip2` package